config/initializers/devise.rb

```diff
@@ -143,7 +143,7 @@
   # without confirming their account.
   # Default is 0.days, meaning the user cannot access the website without
   # confirming their account.
-  config.allow_unconfirmed_access_for = 0.days
```

Comment:
this can be commented out because it's a default value anyways

Reply:
let's have this commented in so it's clear that this is something the user can customise. in the docs we just say that they need to update this so we don't want to make the docs more complex where we need to say to comment it in and then customise or have cause for confusion around this.